tests/kernel/gen_isr_table/src/multilevel_irq.c

```diff
@@ -33,18 +37,26 @@ static void test_multi_level_bit_masks_fn(uint32_t irq1, uint32_t irq2, uint32_t
 		zassert_equal(hwirq1, irq_parent_level(irqn, 2));
 	}
 
-	if (has_l3) {
+#ifdef CONFIG_3RD_LEVEL_INTERRUPTS
```

Comment:
The level 3 APIs weren't guarded in #81318 which caused the compilation [issue](https://github.com/zephyrproject-rtos/zephyr/actions/runs/11924708117/job/33235891540)